src/functional_test_runner/lib/mocha/setup_mocha.js

```diff
@@ -60,8 +60,10 @@ export async function setupMocha(lifecycle, log, config, providers) {
   filterSuitesByTags({
     log,
     mocha,
-    include: config.get('suiteTags.include'),
-    exclude: config.get('suiteTags.exclude'),
+    include: config.get('suiteTags.include')
```

Comment:
Let's also revert this, it's stripping the incremented integer on the end of the job name. If needed @spalger can open another PR for this.